### ISSUE TITLE
rpc with c/c++ backend and browser js frontend

### ISSUE THREAD

**Issue by KrishnaPG** [alvistar/wamped#1]
Good to see another C++11 implementation of WAMP client. The ReadMe indicates a `ws` url. Does this mean, we can use c++ as rpc provider and javascript (from browser) as consumer both talking over websockets?

**Comment by alvistar** [comment]
Yes, as long as you use a router like crossbar in between. To use directly there's need to create a websocket server and wamp broker on the embedded system.

I did that as test in past for a c++ library, but not ported to embedded.

Doable but it will take time from current state.

A.
